Tools/TestWebKitAPI/glib/CMakeLists.txt

```diff
@@ -176,7 +176,6 @@ macro(ADD_WPE_QT_TEST test_name)
     set_target_properties(${test_name} PROPERTIES
         RUNTIME_OUTPUT_DIRECTORY ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/TestWebKitAPI/WPEQt
         AUTOMOC ON
-        CXX_STANDARD 17
```

Comment:
This is perhaps worth calling out in the change log if intentional.

Reply:
Yes, it's intentional. std::span is C++20 only and some of the files in this test include StdLibExtras.h.